RemoteSpiNNaker/RemoteSpiNNakerWeb/src/main/java/uk/ac/manchester/cs/spinnaker/status/StatusCakeStatusMonitorManagerImpl.java

```diff
@@ -73,7 +73,8 @@
      */
     @PostConstruct
     private void init() {
-        final ResteasyClient client = new ResteasyClientBuilder().
+        // TODO use RestClientUtils.createClient()
```

Comment:
I have a feeling that this code ought to use the client utils rather than doing this stuff directly. Not that it's a priority here. (`ResteasyClientBuilder` is an abstract class in Resteasy 4.)

Reply:
It looks like ClientBuilder is now a JaxRS thing (I suspect this has happened between version 3 and 4 or RestEasy), so it looks like the pattern is just to do ClientBuilder.newBuilder()... and then the rest as it is now (see https://docs.jboss.org/resteasy/docs/4.6.0.Final/userguide/html_single/index.html#d4e3448).